src/styles/blocks/calendar.scss

```
@@ -0,0 +1,53 @@
+
+.calendar {
+  display: flex;
+  padding: 50px;
+  margin: auto;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: $calendar-max-width;
+
+  &_day {
+    box-sizing: border-box;
+    display: flex;
+    width: 100px;
+    height: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    transition: background-color transform;
+    transition-duration: 0.5s;
+  }
+
+  &_day:hover {
```

write inside day class
```suggestion

  &:hover {
```